server/api/models/Fundraise.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  attributes: {
+    campaignName: {
+      type: 'string',
+      required: true,
+    },
+    categoryName: {
+      type: 'string',
+      required: true,
+    },
+    categoryKey: {
```

Comment:
whats category key and name for?

Reply:
in giving tuesday, there are categories of giving so we want to account for that. the example would be
```
categoryName: Food for the poor
categoryKey: food-for-the-poor
```